lib/Dialect/TTIR/Pipelines/TTIRPipelines.cpp

```diff
@@ -20,6 +21,7 @@ void createStableHLOToTTIRPipeline(
   if (options.arithDialectConversionsEnabled) {
     pm.addPass(createConvertArithToStableHLOPass());
   }
+  pm.addPass(stablehlo::createStablehloLegalizeCompositeToCallPass());
```

Comment:
Can you please guard it with an option that may be enabled by default? This will allow someone to disable this pass in future if needed without having to rebuild it.

Reply:
Done